oops/backplane/spheroid.py

```diff
@@ -6,16 +6,16 @@
 
 from polymath import Scalar
 
-from .           import Backplane
-from ..constants import PI, TWOPI
+from oops.backplane import Backplane
+from oops.constants import PI, TWOPI
 
-
-############################################################################
+################################################################################
 # Body surface geometry, surface intercept versions
 #   longitude()
 #   latitude()
-############################################################################
+################################################################################
 
+#===============================================================================
```

Comment:
Avoid one horizontal separator immediately after another. It seems unnecessary.

Reply:
Ok I would prpose that the closing ####### separator of the preceding comment block should be removed (here and in general) so that method definitions have a consistent look.

Reply:
I would think the big text block should have `####` before and after, so that it looks like a consistent block (3-sided rectangle). I see no reason to put any `#====` before a function definition. That is not something we do in any other code base.